test/jdk/sun/security/pkcs11/PKCS11Test.java

```diff
@@ -247,6 +247,30 @@ public static String getBase() throws Exception {
         return PKCS11_BASE;
     }
 
+    private static void copyFiles(Path source, String destination)
```

Comment:
Check jdk.test.lib if there is a method you can use to copy files. If not then this method should be added to library.

Reply:
Updated to use the existing Utils.copyFile, it was also needed to add the dependency in a shell test that uses PKCS11Test.java